taskchampion/lib/src/server.rs

```diff
@@ -202,7 +202,7 @@ pub unsafe extern "C" fn tc_server_new_gcp(
                 .to_vec();
             let server_config = ServerConfig::Gcp {
                 bucket,
-                credential_path: Some(credential_path),
+                credential_path: credential_path.map(|cp| cp.to_string()),
```

Comment:
It doesn't look right. What's the type of credential_path?

Reply:
`credential_path` is a String, `credential_path_argument` is a TCString

Reply:
Hm, looking up in this file I see 
```rust
let credential_path = if credential_path == "" {
                None
            } else {
                Some(credential_path)
            };
```
so that's a vlaue of type `Option<String>`. Don't forget that Rust allows new declarations of a variable name with a different type!